tests/cephfs/cephfs_utils.py

```diff
@@ -442,6 +522,29 @@ def fuse_mount(self, fuse_clients, mounting_dir, **kwargs):
                 assert mounting_dir.rstrip("/") in mount_output
                 return 0
 
+    def fuse_mountV1(self, fuse_clients, mounting_dir, **kwargs):
+        for client in fuse_clients:
+            log.info("Creating mounting dir:")
+            client.exec_command(sudo=True, cmd="mkdir %s" % mounting_dir)
+            log.info("Mounting fs with ceph-fuse on client %s:" % client.node.hostname)
+            if kwargs.get("sub_dir"):
+                client.exec_command(
+                    sudo=True,
+                    cmd=f"ceph-fuse -n client.{kwargs.get('new_client_hostname',client.node.hostname)} "
+                    f"{mounting_dir} -r /{kwargs.get('sub_dir')} ",
+                )
+            else:
+                client.exec_command(
+                    sudo=True,
+                    cmd=f"ceph-fuse -n client.{kwargs.get('new_client_hostname', client.node.hostname)} {mounting_dir}",
+                )
```

Comment:
Most part of the code is similar with exception to `-r /{kwargs.get('sub_dir')}` hence could we do something like
```
cmd = f"ceph-fuse -n client.{kwargs.get('new_client_hostname'), client.node.hostname)} {mounting_dir}"
if kwargs.get("sub_dir"):
  cmd += f" -r /{kwargs['sub_dir']"

client.exec_command(sudo=True, cmd=cmd)
```